libs/build_netcdf.sh

```diff
@@ -174,28 +175,31 @@ VERBOSE=$MAKE_VERBOSE make -j1 #NetCDF-Fortran-4.5.2 & intel/20 have a linker bu
 [[ $MAKE_CHECK =~ [yYtT] ]] && make check
 $SUDO make install
 
-cd $curr_dir
+if [ $enable_cxx =~ [yYtT] ]; then 
+   
+  cd $curr_dir
 
-$MODULES || echo $software >> ${HPC_STACK_ROOT}/hpc-stack-contents.log
+  $MODULES || echo $software >> ${HPC_STACK_ROOT}/hpc-stack-contents.log
```

Comment:
can you move this line to after 191.

This is also probably true above in the C and Fortran sections.  Otherwise it is probably just echo-ing C 

After this, all looks good.